roles/openshift_master/tasks/main.yml

```diff
@@ -27,6 +27,9 @@
   action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-master{{ openshift_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
   when: not openshift.common.is_containerized | bool
 
+- name: Install bash completion for oc tools
+  action: "{{ ansible_pkg_mgr }} name=bash-completion state=present"
```

Comment:
kubeconfig is in master only because the admin kubeconfig does not exist on a host that isn't a master, it is a convenience that only makes sense on masters...  having bash-completion on client hosts makes sense on all client hosts :)